intranet/templates/page_with_header.html

```diff
@@ -92,15 +92,25 @@ <h1>Intranet</h1>
                     </div>
                     <ul class="dropdown-menu">
                         <div class="arrow"></div>
-                        <li><a href="{% url 'user_profile'%}"><i class="fas fa-user"></i> My Profile</a></li>
-                        <li><a href="{% url 'preferences' %}"><i class="fas fa-cog"></i> Preferences</a></li>
-                        {% if not request.user.is_restricted %}
-                        <li><a href="{% url 'itemreg' %}"><i class="fas fa-edit"></i> Item Registration</a></li>
-                        <li><a href="{% url 'lostfound' %}"><i class="far fa-compass"></i> Lost and Found</a></li>
-                        <li><a href="{% url 'oauth2_provider:list' %}"><i class="fas fa-key"></i> OAuth</a></li>
-                        {% endif %}
+                        <li>
+                            <div style="float:left;">
+                                <a href="{% url 'user_profile'%}"><i class="fas fa-user"></i> My Profile</a>
+                            </div>
+                            <div class="has-dropdown-subgroup">
+                                <a class="subgroup-arrow-parent"><i class="fas fa-caret-down subgroup-arrow"></i></a>
```

Comment:
```suggestion
                                <a class="subgroup-arrow-parent" href="#"><i class="fas fa-caret-down subgroup-arrow"></i></a>
```

Kind of a nitpick, but if one were to hover over the arrow without this, the cursor would not turn into a "thumb" (or otherwise show as clickable).